content/introduction/getting-started-with-sdks/index.mdx

```diff
@@ -0,0 +1,485 @@
+
+</Steps>
+
+Now you can use `sdkTrusted` to make requests that rely on a secure
+context, e.g. deleting a user with
+[Marketplace API](https://www.sharetribe.com/api-reference/marketplace.html#delete-user).
+
+<Callout type="info">
+  Requests that require a trusted SDK rely on valid login credentials
+  appropriate for the request. For example, to delete a user you must
+  log in with that user's credentials.
+</Callout>
+
+## Example usage
+
+Here are a few examples demonstrating common uses and capabilities of
+the SDKs.
+
+### Logging in
+
+**API**: Marketplace API
+
+Some endpoints require that a user is logged in first, e.g. creating
+listings, viewing their own listings, or initiating inquiry
+transactions.
+
+```jsx
+const sharetribeSdk = require('sharetribe-flex-sdk');
+
+var sdk = sharetribeSdk.createInstance({
+  clientId: '<Client ID>',
+});
+
+sdk
+  .login({ username: '<User email>', password: '<User password>' })
+  .then((_) => {
+    // Request dependent on an authenticated user access token
+  });
+```
+
+### Initiating a transaction
+
+**API**: Marketplace API
+
+The process of initiating a transaction is highly variable because it
+depends entirely on transaction flow customizations and the required
+paramters of the transition.
+
+For this example, we'll use the default-booking process, and choose
+`transition/request-payment` as the first step. This requires that:
+
+- The customer is logged in
+- A token exchange is made to get a trusted user access token
+- The request includes the following parameters
+  - `transition`
+  - `processAlias`
+  - `params`
+    - `listingID`
+    - `bookingStart`
+    - `bookingEnd`
+    - `lineItems`
+
+<Callout type="info">
+  This example uses the **minimum** required parameters per the
+  transition defined in the transaction process. You'll likely want to
+  include more information during this step, e.g. updating the
+  transaction's extended data. See the [Marketplace API reference
+  documentation](https://www.sharetribe.com/api-reference/marketplace.html#initiate-transaction)
+  for more information.
+</Callout>
+
+```jsx
+const transition = 'transition/request-payment';
+const processAlias = 'default-booking/release-1';
+
+const params = {
+  listingId: '<Listing ID>',
+  bookingStart: '2025-12-20T22:00:00.000Z',
+  bookingEnd: '2025-12-21T22:00:00.000Z',
+  lineItems: [
+    {
+      code: 'line-item/night',
+      unitPrice: {
+        _sdkType: 'Money',
+        amount: 5000,
+        currency: 'EUR',
+      },
+      quantity: 1,
+      includeFor: ['customer', 'provider'],
+    },
+    {
+      code: 'line-item/provider-commission',
+      unitPrice: {
+        _sdkType: 'Money',
+        amount: 5000,
+        currency: 'EUR',
+      },
+      percentage: -13,
+      includeFor: ['provider'],
+    },
+  ],
+};
+
+const sdk = sharetribeSdk.createInstance({
+  clientID: '<Client ID>',
+  clientSecret: '<Client secret>',
+  tokenStore: sharetribeSdk.tokenStore.memoryStore(),
```

Comment:
I think you can leave this tokenStore out when you are just initialising the Sharetribe SDK